certificates_test.go

```diff
@@ -0,0 +1,186 @@
+package ydb
```

Comment:
The certificates_test.go is located far from certificates.go
What about move certificates_test.go to internal/certificates?

Reply:
This test is in root package because we need to benchmark and test `connection.With` method because resource usage in this method is the main motivation for the issue. Testing for example just `certificates.ParseCertificatesFromFile` will give much less information.

Reply:
Now find the file is difficult and when I see to certificates.go - I see about it has no tests.

If you want solve problem of cycle import - you can declare package with _test suffix (```package certificates_test```) in certificates_test.go, placed near to certificates.go

Reply:
Renamed test file to `with_test.go`